projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -1078,3 +1078,33 @@ function wpcom_get_launchpad_checklist_title_by_checklist_slug( $checklist_slug
 
 	return wpcom_launchpad_checklists()->get_task_list_title( $checklist_slug );
 }
+
+/**
+ * Gets a launchpad config option.
+ *
+ * @param string $option The option to get.
+ * @param mixed  $default The default value to return if the option is not set.
+ */
+function wpcom_get_launchpad_config_option( $option, $default = null ) {
+	$wpcom_launchpad_config = get_option( 'wpcom_launchpad_config', array() );
+
+	if ( ! isset( $wpcom_launchpad_config[ $option ] ) ) {
```

Comment:
Probably worth rewriting this condition as `! is_array( $wpcom_launchpad_config ) || ! isset( $wpcom_launchpad_config[ $option ] )` as options can be set from anywhere